lib/global-styles.php

```diff
@@ -270,13 +270,13 @@ function gutenberg_experimental_global_styles_get_theme() {
  */
 function gutenberg_experimental_global_styles_get_supported_styles( $supports ) {
 	$style_features = array(
-		'color'            => array( '__experimentalColor' ),
-		'background-color' => array( '__experimentalColor' ),
-		'background'       => array( '__experimentalColor', 'gradients' ),
-		'link-color'       => array( '__experimentalColor', 'linkColor' ),
-		'line-height'      => array( '__experimentalLineHeight' ),
-		'font-size'        => array( '__experimentalFontSize' ),
-		'block-align'      => array( 'align' ),
+		'--wp--style--color--link' => array( '__experimentalColor', 'linkColor' ),
+		'background-color'         => array( '__experimentalColor' ),
+		'background'               => array( '__experimentalColor', 'gradients' ),
+		'block-align'              => array( 'align' ),
```

Comment:
This doesn't make sense for me, though it's not introduced in this particular PR.

Reply:
I've noticed it as well, made a note to review later.

Reply:
Heya!  Sorry, that was my doing... does block align not belong in this function in general?

Its being used for checking if a block as align hook support here:
https://github.com/WordPress/gutenberg/blob/10871217bc48c4f5ded3e13a9088f6ff91da3518/lib/blocks.php#L421

but I can remove it from this function and just check if the block_type has the align flag there instead if that makes more sense?  I currently have #24300 open in this area to add support for default values and can update this there if needed.  (Also I wouldn't mind any input/review on that PR, I didn't realize we missed the case for default values when we originally added those functions in #23007.)

Reply:
Riad already proposed an alternative at https://github.com/WordPress/gutenberg/pull/24351